src/deploy/Configurer.sol

```diff
@@ -253,4 +253,28 @@ contract Configurer {
         master.setBooster(booster);
         master.setDefaultSafeAuthority(defaultAuthority);
     }
+
+    function configureAdmin(MultiRolesAuthority turboAuthority, Comptroller pool, TurboAdmin admin) public {
+        // temporarily assume ownership of pool (required by deployer)
+        pool._acceptAdmin();
+        
+        // Temporarily grant the deployer the turbo admin role for setup
+        turboAuthority.setUserRole(address(this), TURBO_ADMIN_ROLE, true);
+        pool._setPendingAdmin(address(admin));
+        admin._acceptAdmin();
+    }
+    
+    function resetOwnership(
+        MultiRolesAuthority defaultAuthority,
+        MultiRolesAuthority turboAuthority,
+        TimelockController turboTimelock,
+        address admin
+    ) public {
```

Comment:
This function should be permissioned as it sets ownership of the turbo timelock controller.